docs/spanning_tree/4_handling_root_announcements.md

```diff
@@ -29,6 +29,14 @@ The following sanity checks must be performed on all incoming root announcement
 
 If any of these conditions fail, the update is considered to be invalid, the update should be dropped and the peer that sent us this announcement should be disconnected as a result of the error.
 
+## Storing root announcements
+
+When storing a root announcement for a given peer, the following information should be kept:
+
+- The announcement itself;
+- The time the announcement was received;
+- The order in which the announcement was received;
```

Comment:
> The order in which the announcement was received

This is a bit vague so we might want to make clearer that we're actually concerned about global ordering *across all peers* — in effect, we need to be able to tell whether this peer's update arrived before or after any other given peer. A global atomic counter is one way to do that (which is what we do in `s._ordering`).